### PR TITLE
Restructure URLs learn.greensofware.foundation instead of learn.greensofware.foundation/practitioner/introduction

### PR DESCRIPTION
**Pull request by jawache**
- Moved the practitioner folder to the root, so now this is just for practitioner training.
- Added redirects so anyone heading to the old links will be redirected to the new links.
- Added a new landing page with more course like landing page copy.

### CODE REVIEW

**Review by greenhsu123 — COMMENTED** [review]
lgtm! Thanks!

File: docs/index.mdx [hunk 1]
```diff
@@ -2,14 +2,42 @@
...
+- Have at least **one year of industry experience** building, deploying, or managing software applications.
+- Complete each section in order from start to finish.
+- Complete the knowledge checks at the end of each section.
+- If you would like to gain a certificate of completion, take the exam through the Linux Foundation.
+
+## By the end of this training, you will understand the following:
+
+- The core drivers of carbon emissions for software applications.
+- The 3 actions you can take to reduce the carbon emissions of your software.
+- How to measure software emissions using the Greenhouse Gas (GHG) or the Software Carbon Intensity (SCI) protocols.
```
Comment (greenhsu123) [inline docs/index.mdx:36]: Just checking that we also want to use the word "protocols" for SCI, not specification?

**Review by greenhsu123 — COMMENTED** [review]
lgtm!

File: docs/index.mdx (hunk 1 above)
Comment (jawache) [reply docs/index.mdx]: Good catch, updated :)

**Comment by jawache** [comment]
@greenhsu123 updated as per comments :)

**Comment by jawache** [comment]
If you can merge sooner rather than later would be good, want to make sure the redirects work as expected.

**Pull request merged**